examples/tensorflow-akka/build.sbt

```diff
@@ -0,0 +1,53 @@
+//tag::docs-projectSetup-example[]
+import sbt._
+import sbt.Keys._
+
+import scalariform.formatter.preferences._
+
+lazy val tensorflowAkka =  (project in file("."))
+    .enablePlugins(CloudflowAkkaStreamsApplicationPlugin)
+    .settings(
+//end::docs-projectSetup-example[]
+      libraryDependencies ++= Seq(
+        "ch.qos.logback"         %  "logback-classic"           % "1.2.3",
+        "com.typesafe.akka"      %% "akka-http-testkit"         % "10.1.10" % "test",
+        "org.tensorflow"         % "tensorflow"                 % "1.13.1",
```

Comment:
Could you try with version 1.15.0 which is the latest?

Reply:
I think the example will need to be updated soon: https://github.com/tensorflow/tensorflow/issues/32943

Reply:
I'll more to 1.15.0 for now.